packages/slate-editor/src/modules/editor/getEnabledExtensions.ts

```diff
@@ -235,8 +235,8 @@ export function* getEnabledExtensions(parameters: Parameters): Generator<Extensi
     if (withGalleries) {
         yield GalleriesExtension({
             availableWidth,
-            onEdited(editor, gallery, { failedUploads }) {
-                EventsEditor.dispatchEvent(editor, 'gallery-edited', {
+            onAdded(editor, gallery, { failedUploads }) {
```

Comment:
Since the behaviour has changed, "add" will now only append images to the end of the gallery instead of editing the whole gallery.

Do you think it's worth keeping the "onEdited" callback/event in addition to having the `onAdd/onAdded` callback?

`onEdited` would still receive the same data as before.